diff --git a/node_modules/webpack-webext-plugin/index.js b/node_modules/webpack-webext-plugin/index.js
index 7b91047..0380b56 100644
--- a/node_modules/webpack-webext-plugin/index.js
+++ b/node_modules/webpack-webext-plugin/index.js
@@ -60,7 +60,7 @@ WebpackWebExt.prototype.runWebExtCommand = function() {
 
 WebpackWebExt.prototype.apply = function(compiler) {
   compiler.plugin('emit', (compilation, callback) => {
-    if (this.runOnce && !this.webExtRunPromise) {
+    if (this.runOnce) {
       if (!this.webExtRunPromise) {
         this.webExtRunPromise = this.runWebExtCommand();
       }
